vite.config.ts

```diff
@@ -15,6 +15,7 @@ export default defineConfig(({ mode }) => {
   const outDir = 'dist'
 
   const forceLocalStorage = env.FORCE_LOCAL_STORAGE === 'true'
+  const privateJrPlatform = env.PRIVATE_JR_PLATFORM === 'true'
```

Comment:
* [ ] FYI I naively tried with `PRIVATE_JR_PLATFORM=1` and wondered why it didn't work 🙈